scripts/st2bootstrap-el8.template.sh

```diff
@@ -145,15 +153,21 @@ install_st2_dependencies() {
 }
 
 install_mongodb() {
-  # Add key and repo for the latest stable MongoDB (3.4)
-  sudo rpm --import https://www.mongodb.org/static/pgp/server-3.4.asc
-  sudo sh -c "cat <<EOT > /etc/yum.repos.d/mongodb-org-3.4.repo
-[mongodb-org-3.4]
+
+  # Need yum perl module enabled on RHEL 8
```

Comment:
Can you please clarify this part about yum perl module, which package or component requires it?

Reply:
@armab When installing rabbitmq, on RHEL, I was getting this error: 

```
 From       : https://packagecloud.io/rabbitmq/rabbitmq-server/gpgkey
Generating yum cache for rabbitmq_rabbitmq-server-source...

The repository is setup! You can now install packages.
Updating Subscription Management repositories.
rabbitmq_rabbitmq-server                        548  B/s | 833  B     00:01    
Modular dependency problems:

 Problem 1: conflicting requests
  - nothing provides module(perl:5.26) needed by module perl-DBD-SQLite:1.58:8010020190322125518:073fa5fe-0.x86_64
 Problem 2: conflicting requests
  - nothing provides module(perl:5.26) needed by module perl-DBI:1.641:8010020190322130042:16b3ab4d-0.x86_64
Metadata cache created.
Updating Subscription Management repositories.
```

The perl modules resolve it. The reason I installed it here, is because I was trying to get them enabled as early as possible.

Reply:
I have moved this to where it should have been (right before rabbitmq install) and explained the need.

Reply:
@armab I just double checked this and tested it, and the perl modules are needed for RHEL